### DISCUSSION TITLE
any idea on when composer 2 will be supported

### DISCUSSION THREAD

**Discussion by steveb85** [uvdesk/community-skeleton#536] [Ideas]
composer 1 is depreciated and we have other software on or server requesting composer 2, so wondering when it will be supported.

**Comment by PeopleInside** [comment]
Wht you think composer 2 is not supported?
I use composer 2 and not composer 1

**Reply by steveb85** [reply]
ohh I had believed only 1 was supported and had been holding off upgrading for some time, this mst be my mistake! I’ll be upgrading asap! thanks Marco!

**Reply by PeopleInside** [reply]
You are welcome ;)
Made always a full folder and database backup before do any upgrading action.
